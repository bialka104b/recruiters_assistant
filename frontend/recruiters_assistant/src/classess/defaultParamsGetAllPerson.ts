export class Params {
	surname: string;
	firstname: string;
	locality: string;
	specjalization: string;
	arrayTechnologies: Array<string>;
	adobe: string;
	agile: string;
	android: string;
	angular: string;
	aws: string;
	bash: string;
	bootstrap: string;
	css: string;
	csharp: string;
	cpp: string;
	c: string;
	delphi: string;
	html: string;
	ios: string;
	java: string;
	javascript: string;
	jQuery: string;
	kanban: string;
	less: string;
	linux: string;
	dotNet: string;
	node: string;
	oracle: string;
	perl: string;
	photoshop: string;
	php: string;
	powershell: string;
	python: string;
	react: string;
	ruby: string;
	sass: string;
	scala: string;
	scrum: string;
	spring: string;
	sql: string;
	scss: string;
	swift: string;
	vue: string;
	vb: string;
	windows: string;
	angielski: string;
	angielskia2: string;
	angielskib1: string;
	angielskib2: string;
	angielskic1: string;
	angielskic2: string;
	niemiecki: string;
	niemieckia2: string;
	niemieckib1: string;
	niemieckib2: string;
	niemieckic1: string;
	niemieckic2: string;
	pozostaleJezyki: string;
	constructor(
		surname: string = "",
		firstname: string = "",
		locality: string = "",
		specjalization: string = "",
		arrayTechnologies: Array<string> = [],
		adobe: string = "",
		agile: string = "",
		android: string = "",
		angular: string = "",
		aws: string = "",
		bash: string = "",
		bootstrap: string = "",
		css: string = "",
		csharp: string = "",
		cpp: string = "",
		c: string = "",
		delphi: string = "",
		html: string = "",
		ios: string = "",
		java: string = "",
		javascript: string = "",
		jQuery: string = "",
		kanban: string = "",
		less: string = "",
		linux: string = "",
		dotNet: string = "",
		node: string = "",
		oracle: string = "",
		perl: string = "",
		photoshop: string = "",
		php: string = "",
		powershell: string = "",
		python: string = "",
		react: string = "",
		ruby: string = "",
		sass: string = "",
		scala: string = "",
		scrum: string = "",
		spring: string = "",
		sql: string = "",
		scss: string = "",
		swift: string = "",
		vue: string = "",
		vb: string = "",
		windows: string = "",
		angielski: string = "",
		angielskia2: string = "",
		angielskib1: string = "",
		angielskib2: string = "",
		angielskic1: string = "",
		angielskic2: string = "",
		niemiecki: string = "",
		niemieckia2: string = "",
		niemieckib1: string = "",
		niemieckib2: string = "",
		niemieckic1: string = "",
		niemieckic2: string = "",
		pozostaleJezyki: string = "",
	) {
		this.surname = surname;
		this.firstname = firstname;
		this.locality = locality;
		this.specjalization = specjalization;
		this.arrayTechnologies = arrayTechnologies;
		this.adobe = adobe;
		this.agile = agile;
		this.android = android;
		this.angular = angular;
		this.aws = aws;
		this.bash = bash;
		this.bootstrap = bootstrap;
		this.css = css;
		this.csharp = csharp;
		this.cpp = cpp;
		this.c = c;
		this.delphi = delphi;
		this.html = html;
		this.ios = ios;
		this.java = java;
		this.javascript = javascript;
		this.jQuery = jQuery;
		this.kanban = kanban;
		this.less = less;
		this.linux = linux;
		this.dotNet = dotNet;
		this.node = node;
		this.oracle = oracle;
		this.perl = perl;
		this.photoshop = photoshop;
		this.php = php;
		this.powershell = powershell;
		this.python = python;
		this.react = react;
		this.ruby = ruby;
		this.sass = sass;
		this.scala = scala;
		this.scrum = scrum;
		this.spring = spring;
		this.sql = sql;
		this.scss = scss;
		this.swift = swift;
		this.vue = vue;
		this.vb = vb;
		this.windows = windows;
		this.angielski = angielski;
		this.angielskia2 = angielskia2;
		this.angielskib1 = angielskib1;
		this.angielskib2 = angielskib2;
		this.angielskic1 = angielskic1;
		this.angielskic2 = angielskic2;
		this.niemiecki = niemiecki;
		this.niemieckia2 = niemieckia2;
		this.niemieckib1 = niemieckib1;
		this.niemieckib2 = niemieckib2;
		this.niemieckic1 = niemieckic1;
		this.niemieckic2 = niemieckic2;
		this.pozostaleJezyki = pozostaleJezyki;
	}
}
