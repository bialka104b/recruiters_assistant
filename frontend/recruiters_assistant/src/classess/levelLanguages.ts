export class LevelObject {
	level: string;
	objectArr: Array<string>;

	constructor(level: string = "Clear", objectArr: Array<string> = []) {
		this.level = level;
		this.objectArr = objectArr;
	}
}
