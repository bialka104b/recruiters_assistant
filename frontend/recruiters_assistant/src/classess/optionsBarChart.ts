export class Options {
	responsive: boolean;
	plugins: object;
	maintainAspectRatio: boolean;
	constructor() {
		(this.responsive = true),
			(this.maintainAspectRatio = false),
			(this.plugins = {
				tooltip: {
					callbacks: {
						label: function (context: any) {
							let label = context.dataset.label || "";

							if (label && context.parsed.y !== null) {
								label += ": ";
								if (context.parsed.y) {
									label += `${context.parsed.y} w wieku ${context.label} lat`;
								}
							} else {
								label += `: ${context.parsed.y}`;
							}
							return label;
						},
					},
				},
			});
	}
}
