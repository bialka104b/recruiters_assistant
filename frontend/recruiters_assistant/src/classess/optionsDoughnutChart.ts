export class OptionsDonut {
	responsive: boolean;
	plugins: object;
	maintainAspectRatio: boolean;
	constructor() {
		(this.responsive = true),
			(this.maintainAspectRatio = false),
			(this.plugins = {
				tooltip: {
					callbacks: {
						label: function (context: any) {
							let label = context.label || "";

							if (label && context.parsed !== null) {
								label += ": ";
								const anyString = context.parsed.toString();
								if (context.parsed == 1) {
									label += `${context.parsed} osoba`;
								} else if (
									["2", "3", "4"].includes(
										anyString.substring(anyString.length - 1),
									) &&
									!["12", "13", "14"].includes(anyString)
								) {
									label += `${context.parsed} osoby`;
								} else {
									label += `${context.parsed} os√≥b`;
								}
							}
							return label;
						},
					},
				},
			});
	}
}
