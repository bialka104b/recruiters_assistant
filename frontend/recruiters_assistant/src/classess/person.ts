export class Person {
	Angielski: String;
	Data_kontaktu: String;
	Doswiadczenie: String;
	Email:  String;
	Firmy_Wspolpraca:  String;
	Imie: String;
	Komentarze:  String;
	Link_Do_Profilu: String;
	Miejscowosc: String;
	Nazwisko: String;
	Niemiecki:  String;
	Pozostale_Jezyki:  String;
	Relokacja:  String;
	Rozmowa_nietechniczna:  String;
	Rozmowa_techniczna:  String;
	Specjalnosc:  String;
	Status_Zainteresowany: Boolean
	Technologie: String;
	Telefon: String;
	Wiek:  String;
	Wym_Finansowe:  String;
	_id: String;

	constructor(
		_id: String = '',
		Imie: String = '',
		Nazwisko: String = '',
		Angielski: String = '',
		Niemiecki:  String = '',
		Doswiadczenie: String = '',
		Email:  String = '',
		Firmy_Wspolpraca:  String = '',
		Komentarze:  String = '',
		Link_Do_Profilu: String = '',
		Miejscowosc: String = '',
		Pozostale_Jezyki:  String = '',
		Relokacja:  String = '',
		Rozmowa_nietechniczna:  String = '',
		Rozmowa_techniczna:  String = '',
		Specjalnosc:  String = '',
		Status_Zainteresowany: Boolean = false,
		Technologie: String = '',
		Telefon: String = '',
		Wym_Finansowe:  String = '',
		Data_kontaktu: String = '',
		Wiek:  String = '',
	) { 
		this.Angielski = Angielski,
		this.Data_kontaktu = Data_kontaktu,
		this.Doswiadczenie = Doswiadczenie,
		this.Email = Email,
		this.Firmy_Wspolpraca = Firmy_Wspolpraca,
		this.Imie= Imie,
		this.Komentarze = Komentarze,
		this.Link_Do_Profilu =Link_Do_Profilu,
		this.Miejscowosc =Miejscowosc,
		this.Nazwisko =Nazwisko,
		this.Niemiecki = Niemiecki,
		this.Pozostale_Jezyki = Pozostale_Jezyki,
		this.Relokacja = Relokacja,
		this.Rozmowa_nietechniczna = Rozmowa_nietechniczna,
		this.Rozmowa_techniczna = Rozmowa_techniczna,
		this.Specjalnosc = Specjalnosc,
		this.Status_Zainteresowany = Status_Zainteresowany,
		this.Technologie =Technologie,
		this.Telefon =Telefon,
		this.Wiek = Wiek,
		this.Wym_Finansowe = Wym_Finansowe,
		this._id = _id
	}
}