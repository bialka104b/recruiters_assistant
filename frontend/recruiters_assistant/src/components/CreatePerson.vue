<template v-if="showModalCreate">
	<div>
		<!-- this code is for unit testing  -->
		<!-- <button class="mx-1" @click="createPerson(candidate)">Zapisz</button>
		<button class="mx-1" @click="cancel()">Cancel</button>
		<input
			class="mt-2 wrapperInput"
			v-model="candidate.Imie"
			label="Imię:"
			input-class="inputClass"
		/> -->
		<va-modal
			v-model="showModalCreate"
			:no-dismiss="true"
			:hide-default-actions="true"
			:blur="false"
			fullscreen
			:fixed-layout="true"
		>
			<template #header>
				<div class="col-12 col-md-12">
					<h3>Dodaj kandydata: {{ candidate.Imie }} {{ candidate.Nazwisko }}</h3>
				</div>
			</template>
			<template #default>
				<va-form class="row" tag="form">
					<div class="opakowanie col-md-6">
						<va-input
							class="mt-2 wrapperInput"
							v-model="candidate.Imie"
							label="Imię:"
							input-class="inputClass"
						/>
					</div>
					<div class="opakowanie col-md-6">
						<va-input
							class="mt-2 wrapperInput"
							v-model="candidate.Nazwisko"
							label="Nazwisko:"
							input-class="inputClass"
						/>
					</div>
					<div class="opakowanie col-md-6">
						<va-input
							class="mt-2 wrapperInput"
							v-model="candidate.Email"
							label="Email:"
							input-class="inputClass"
						/>
					</div>
					<div class="opakowanie col-md-6">
						<va-input
							class="mt-2 wrapperInput"
							v-model="candidate.Telefon"
							label="Telefon:"
							input-class="inputClass"
						/>
					</div>
					<div class="opakowanie col-md-6">
						<va-input
							class="mt-2 wrapperInput"
							v-model="candidate.Miejscowosc"
							label="Miejscowosc:"
							input-class="inputClass"
						/>
					</div>
					<div class="opakowanie col-md-6">
						<va-input
							class="mt-2 wrapperInput"
							v-model="candidate.Angielski"
							label="Angielski:"
							input-class="inputClass"
						/>
					</div>
					<div class="opakowanie col-md-6">
						<va-input
							class="mt-2 wrapperInput"
							v-model="candidate.Niemiecki"
							label="Niemiecki:"
							input-class="inputClass"
						/>
					</div>
					<div class="opakowanie col-md-6">
						<va-input
							class="mt-2 wrapperInput"
							v-model="candidate.Pozostale_Jezyki"
							label="Inny język:"
							input-class="inputClass"
						/>
					</div>
					<div class="opakowanie col-md-6">
						<va-input
							class="mt-2 wrapperInput"
							v-model="candidate.Wiek"
							label="Wiek:"
							input-class="inputClass"
						/>
					</div>
					<div class="opakowanie col-md-6">
						<va-input
							class="mt-2 wrapperInput"
							v-model="candidate.Relokacja"
							label="Relokacja:"
							input-class="inputClass"
						/>
					</div>
					<div class="opakowanie col-md-6">
						<va-input
							class="mt-2 wrapperInput"
							v-model="candidate.Doswiadczenie"
							label="Doswiadczenie:"
							input-class="inputClass"
						/>
					</div>
					<div class="opakowanie col-md-6">
						<va-input
							class="mt-2 wrapperInput"
							v-model="candidate.Wym_Finansowe"
							label="Wymagania Finansowe:"
							input-class="inputClass"
						/>
					</div>
					<div class="opakowanie col-md-6">
						<va-date-input
							class="mt-2 wrapperInput"
							v-model="candidate.Data_kontaktu"
							label="Data kontaktu"
						/>
					</div>
					<div class="opakowanie col-md-6">
						<va-input
							class="mt-2 wrapperInput"
							v-model="candidate.Link_Do_Profilu"
							label="Link do profilu:"
							input-class="inputClass"
						/>
					</div>
					<div class="opakowanie col-md-12">
						<va-input
							class="mt-2 wrapperInput"
							v-model="candidate.Technologie"
							label="Technologie:"
							input-class="inputClass"
						/>
					</div>
					<div class="opakowanie col-md-12">
						<va-input
							class="mt-2 wrapperInput"
							v-model="candidate.Specjalnosc"
							label="Specjalnosc:"
							input-class="inputClass"
						/>
					</div>
					<div class="opakowanie col-md-12">
						<va-input
							class="mt-2 wrapperInput"
							v-model="candidate.Firmy_Wspolpraca"
							label="Współpraca z firmami:"
							input-class="inputClass"
						/>
					</div>
					<div class="opakowanie col-md-12">
						<va-input
							class="mt-2 wrapperInput textArea"
							v-model="candidate.Komentarze"
							label="Komentarze:"
							input-class="inputClass"
							type="textarea"
							autosize
						/>
					</div>
					<div class="opakowanie col-md-6">
						<va-input
							class="mt-2 wrapperInput"
							v-model="candidate.Rozmowa_techniczna"
							label="Rozmowa techniczna:"
							input-class="inputClass"
						/>
					</div>
					<div class="opakowanie col-md-6">
						<va-input
							class="mt-2 wrapperInput"
							v-model="candidate.Rozmowa_nietechniczna"
							label="Rozmowa nietechniczna:"
							input-class="inputClass"
						/>
					</div>
				</va-form>
			</template>
			<template #footer>
				<va-button class="mx-1 create" @click="createPerson(candidate)">Zapisz</va-button>
				<va-button class="mx-1 cancel" @click="cancel()">Cancel</va-button>
			</template>
		</va-modal>
	</div>
</template>

<script lang="ts">
import { defineComponent } from "vue";
import Axios from "axios";
import { NewPerson } from "../classess/newPerson";

export default defineComponent({
	name: "CreatePerson",
	emits: ["closeCreateModal", "successCreate"],
	props: {
		showModalCreate: Boolean,
	},
	data() {
		return {
			candidate: new NewPerson(),
		};
	},
	created() {},
	methods: {
		createPerson(candidate: NewPerson) {
			this.create(candidate);
			this.$emit("closeCreateModal", this.showModalCreate);
			this.$emit("successCreate", this.showModalCreate);
			this.candidate = new NewPerson();
		},
		cancel() {
			this.$emit("closeCreateModal", this.showModalCreate);
			this.candidate = new NewPerson();
		},

		async create(candidate: NewPerson) {
			const config = {
				method: "post",
				url: `http://localhost:8080/kandydaci/`,
				headers: {
					"Content-Type": "application/json",
				},
				data: candidate,
			};
			await Axios(config)
				.then((res) => {
					console.log(res, "res udało sie create");
				})
				.catch((err) => {
					console.log(err, "error mmmmmmm create");
				});
		},
	},
});
</script>
