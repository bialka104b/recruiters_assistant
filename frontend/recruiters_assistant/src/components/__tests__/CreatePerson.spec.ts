import { describe, it, expect, test } from "vitest";

import { mount, shallowMount } from "@vue/test-utils";
import CreatePerson from "../CreatePerson.vue";
import { Person } from "@/classess/person";

test("CreatePerson", () => {
	const buttonSelectorCancel = ".cancel";
	const buttonSelectorDelete = ".create";
	const person = new Person();
	const wrapper = shallowMount(CreatePerson, {
		props: {
			person: person,
			showModalDelete: true,
		},
		global: {
			stubs: {
				"va-modal": {
					template: "<div></div>",
				},
				"va-button": {
					template: "<div></div>",
				},
				"va-form": {
					template: "<div></div>",
				},
				"va-date-input": {
					template: "<div></div>",
				},
				"va-input": {
					template: "<div></div>",
				},
				"va-file-upload": {
					template: "<div></div>",
				},
				"va-pagination": {
					template: "<div></div>",
				},
				"va-checkbox": {
					template: "<div></div>",
				},
			},
		},
	});
	
	it("processes valid props data", async () => {
		setTimeout(async () => {
			expect(wrapper.findAll("h2").length).toEqual(1);

			expect(wrapper.findAll("h2").at(0)?.text()).toMatch(person.Imie);
		}, 100);
	});

	it("processes valid props data", async () => {
		setTimeout(async () => {
			expect(wrapper.findAll("h2").at(0)?.text()).toMatch(person.Nazwisko);

			expect(await wrapper.find(buttonSelectorCancel).trigger("click")).toBeTruthy();

			expect(await wrapper.find(buttonSelectorDelete).trigger("click")).toBeTruthy();
		}, 100);
	});
	
	it("should work", (ctx) => {
		console.log(ctx.meta.suite.name, "- komponent CreatePerson pracuje");
	});
	
	
	it("renders properly", async () => {
		await wrapper.get("button").trigger("click");
		await wrapper.get("input").trigger("input");
		expect(wrapper.html()).toMatchSnapshot();
	});


	expect(wrapper.html()).toMatchSnapshot();
});
