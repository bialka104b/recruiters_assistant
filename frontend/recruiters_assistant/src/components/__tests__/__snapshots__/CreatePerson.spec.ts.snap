// Vitest Snapshot v1

exports[`CreatePerson > renders properly 1`] = `
"<div person=\\"[object Object]\\" showmodaldelete=\\"true\\">
  <!-- this code is for unit testing  --><button class=\\"mx-1\\">Zapisz</button><button class=\\"mx-1\\">Cancel</button><input class=\\"mt-2 wrapperInput\\" label=\\"Imię:\\" input-class=\\"inputClass\\">
  <va-modal-stub modelvalue=\\"false\\" no-dismiss=\\"true\\" hide-default-actions=\\"true\\" fullscreen=\\"\\" fixed-layout=\\"true\\"></va-modal-stub>
</div>"
`;

exports[`CreatePerson 1`] = `
"<div person=\\"[object Object]\\" showmodaldelete=\\"true\\">
  <!-- this code is for unit testing  --><button class=\\"mx-1\\">Zapisz</button><button class=\\"mx-1\\">Cancel</button><input class=\\"mt-2 wrapperInput\\" label=\\"Imię:\\" input-class=\\"inputClass\\">
  <div modelvalue=\\"false\\" no-dismiss=\\"true\\" hide-default-actions=\\"true\\" fullscreen=\\"\\" fixed-layout=\\"true\\"></div>
</div>"
`;

exports[`mount component 1`] = `
"<div value=\\"test\\">
  <!-- this code is for unit testing  --><button class=\\"mx-1\\">Zapisz</button><button class=\\"mx-1\\">Cancel</button><input class=\\"mt-2 wrapperInput\\" label=\\"Imię:\\" input-class=\\"inputClass\\">
  <va-modal modelvalue=\\"false\\" no-dismiss=\\"true\\" hide-default-actions=\\"true\\" fullscreen=\\"\\" fixed-layout=\\"true\\">
    <va-form class=\\"row\\" tag=\\"form\\">
      <div class=\\"opakowanie col-md-6\\">
        <va-input-stub class=\\"mt-2 wrapperInput\\" modelvalue=\\"\\" label=\\"Imię:\\" input-class=\\"inputClass\\"></va-input-stub>
      </div>
      <div class=\\"opakowanie col-md-6\\">
        <va-input-stub class=\\"mt-2 wrapperInput\\" modelvalue=\\"\\" label=\\"Nazwisko:\\" input-class=\\"inputClass\\"></va-input-stub>
      </div>
      <div class=\\"opakowanie col-md-6\\">
        <va-input-stub class=\\"mt-2 wrapperInput\\" modelvalue=\\"\\" label=\\"Email:\\" input-class=\\"inputClass\\"></va-input-stub>
      </div>
      <div class=\\"opakowanie col-md-6\\">
        <va-input-stub class=\\"mt-2 wrapperInput\\" modelvalue=\\"\\" label=\\"Telefon:\\" input-class=\\"inputClass\\"></va-input-stub>
      </div>
      <div class=\\"opakowanie col-md-6\\">
        <va-input-stub class=\\"mt-2 wrapperInput\\" modelvalue=\\"\\" label=\\"Miejscowosc:\\" input-class=\\"inputClass\\"></va-input-stub>
      </div>
      <div class=\\"opakowanie col-md-6\\">
        <va-input-stub class=\\"mt-2 wrapperInput\\" modelvalue=\\"\\" label=\\"Angielski:\\" input-class=\\"inputClass\\"></va-input-stub>
      </div>
      <div class=\\"opakowanie col-md-6\\">
        <va-input-stub class=\\"mt-2 wrapperInput\\" modelvalue=\\"\\" label=\\"Niemiecki:\\" input-class=\\"inputClass\\"></va-input-stub>
      </div>
      <div class=\\"opakowanie col-md-6\\">
        <va-input-stub class=\\"mt-2 wrapperInput\\" modelvalue=\\"\\" label=\\"Inny język:\\" input-class=\\"inputClass\\"></va-input-stub>
      </div>
      <div class=\\"opakowanie col-md-6\\">
        <va-input-stub class=\\"mt-2 wrapperInput\\" modelvalue=\\"\\" label=\\"Wiek:\\" input-class=\\"inputClass\\"></va-input-stub>
      </div>
      <div class=\\"opakowanie col-md-6\\">
        <va-input-stub class=\\"mt-2 wrapperInput\\" modelvalue=\\"\\" label=\\"Relokacja:\\" input-class=\\"inputClass\\"></va-input-stub>
      </div>
      <div class=\\"opakowanie col-md-6\\">
        <va-input-stub class=\\"mt-2 wrapperInput\\" modelvalue=\\"\\" label=\\"Doswiadczenie:\\" input-class=\\"inputClass\\"></va-input-stub>
      </div>
      <div class=\\"opakowanie col-md-6\\">
        <va-input-stub class=\\"mt-2 wrapperInput\\" modelvalue=\\"\\" label=\\"Wymagania Finansowe:\\" input-class=\\"inputClass\\"></va-input-stub>
      </div>
      <div class=\\"opakowanie col-md-6\\">
        <va-date-input class=\\"mt-2 wrapperInput\\" modelvalue=\\"\\" label=\\"Data kontaktu\\"></va-date-input>
      </div>
      <div class=\\"opakowanie col-md-6\\">
        <va-input-stub class=\\"mt-2 wrapperInput\\" modelvalue=\\"\\" label=\\"Link do profilu:\\" input-class=\\"inputClass\\"></va-input-stub>
      </div>
      <div class=\\"opakowanie col-md-12\\">
        <va-input-stub class=\\"mt-2 wrapperInput\\" modelvalue=\\"\\" label=\\"Technologie:\\" input-class=\\"inputClass\\"></va-input-stub>
      </div>
      <div class=\\"opakowanie col-md-12\\">
        <va-input-stub class=\\"mt-2 wrapperInput\\" modelvalue=\\"\\" label=\\"Specjalnosc:\\" input-class=\\"inputClass\\"></va-input-stub>
      </div>
      <div class=\\"opakowanie col-md-12\\">
        <va-input-stub class=\\"mt-2 wrapperInput\\" modelvalue=\\"\\" label=\\"Współpraca z firmami:\\" input-class=\\"inputClass\\"></va-input-stub>
      </div>
      <div class=\\"opakowanie col-md-12\\">
        <va-input-stub class=\\"mt-2 wrapperInput textArea\\" modelvalue=\\"\\" label=\\"Komentarze:\\" input-class=\\"inputClass\\" type=\\"textarea\\" autosize=\\"\\"></va-input-stub>
      </div>
      <div class=\\"opakowanie col-md-6\\">
        <va-input-stub class=\\"mt-2 wrapperInput\\" modelvalue=\\"\\" label=\\"Rozmowa techniczna:\\" input-class=\\"inputClass\\"></va-input-stub>
      </div>
      <div class=\\"opakowanie col-md-6\\">
        <va-input-stub class=\\"mt-2 wrapperInput\\" modelvalue=\\"\\" label=\\"Rozmowa nietechniczna:\\" input-class=\\"inputClass\\"></va-input-stub>
      </div>
    </va-form>
  </va-modal>
</div>"
`;
